### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Create Web Site_Keyboard: "Settings", Bindings" and "Application Pool" page tabs are not accessible using keyboard.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#127]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated:** MAS2.1.1

**Repro Steps:** 
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "Create Web Site" expandable button and Press Enter.
6. Verify that page tabs on the screen are accessible using keyboard.

**Actual Result:**
"Settings", Bindings" and "Application Pool" page tabs are not accessible using keyboard.

**Expected Result:**
Keyboard focus is not moving to the "Settings", Bindings" and "Application Pool" page tabs and hence they are not accessible through keyboard.

**User Impact:**
Keyboard only users will be unable to navigate among the different page tabs ("Settings", Bindings" and "Application Pool"). Hence the controls under those page tabs will also not be accessible using keyboard.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment:**
[Bug 127_page tabs are not accessible using the keyboard.pptx](https://github.com/Microsoft/IIS.WebManager/files/2327204/Bug.127_page.tabs.are.not.accessible.using.the.keyboard.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
